### PR TITLE
Created : grid component

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
Grid component
Hey I have created  Grid component  in component library. So ignore other files. Focus only on the grid.html , grid.css (in list folder ) , (in components) .

What not to check
Responsiveness

image

### CODE REVIEW

**Review by archit7448 — APPROVED** [review]
good work! looks good to merge

**Review by Megha-Joshi — APPROVED** [review]
Looks good. You can merge it

File: components/grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
+@import url("/index.css");
+@import url("/components/sidebar.css");
+
+body {
+    background-image: none;
+}
+
+.component-container {
+    margin-left: 5rem;
+
```
Comment (zoyatanweer) [inline components/grid/grid.css]: remove spacing

File: components/grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    font-size: 1.5rem;
+}
+.grid-container {
+    height: auto;
+    width: 95%;
+    padding: 1rem;
+    margin-bottom: 4rem;
+    margin-top: 2rem;
+    border: 2px solid var(--border-color); 
+
```
Comment (zoyatanweer) [inline components/grid/grid.css]: remove spacing

File: components/grid/grid.css [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    width: 100%;
+    height: 40vh;
+    padding: 10px;
+    border: 1px solid var( --border-color);
+    display: grid;
+    grid-template-rows: 1fr 1fr 1fr ;
+    grid-template-columns: repeat(7, 1fr) ;
+    grid-gap: 1rem;
+    text-align: center;
+}
```
Comment (zoyatanweer) [inline components/grid/grid.css]: you can make a common class for grid-area-1 and grid-area-2 and in that use those properties which are common in both of them. Here I can see width, padding, border etc are common.

File: components/grid/grid.html [hunk 4]
```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Neos-UI | Grid</title>
+    <link rel="stylesheet" href="grid.css">
+    <script src="https://kit.fontawesome.com/0b2d218a52.js" crossorigin="anonymous"></script>
```
Comment (zoyatanweer) [inline components/grid/grid.html:10]: you can write a comment here mentioning the link which is written.

**Review by zoyatanweer — APPROVED** [review]
- I like the way how your code is structured, nice use of comments.
- nice UI
- clean and easy to understand code
- good to merge with the suggested changes. Great work !

File: components/grid/grid.html (hunk 4 above)
Comment (Shubhamkr2610) [reply components/grid/grid.html:10]: okay , Will do it

File: components/grid/grid.css (hunk 3 above)
Comment (Shubhamkr2610) [reply components/grid/grid.css]: I will refactor  it

File: components/grid/grid.css (hunk 1 above)
Comment (Shubhamkr2610) [reply components/grid/grid.css]: fixed it . you can check now

File: components/grid/grid.css (hunk 2 above)
Comment (Shubhamkr2610) [reply components/grid/grid.css]: fixed

File: components/grid/grid.css (hunk 3 above)
Comment (Shubhamkr2610) [reply components/grid/grid.css]: made the changes

**Pull request merged**